doc/release/README.multilocale.rst

```diff
@@ -0,0 +1,233 @@
+============================
+Multilocale Chapel Execution
+============================
+
+To run Chapel on multiple locales, the following steps are required:
+
+
+0. The following information describes a fairly generic use of
+   multi-locale Chapel, suitable for networked workstations and
+   generic parallel platforms.  For other platforms, refer to the
+   platform-specific READMEs for additional tips.
+
+
+#. Set your ``CHPL_COMM`` environment variable to "gasnet".  For example:
+
+   .. code-block:: bash
+
+     export CHPL_COMM=gasnet
+
+   This specifies that you wish to use the GASNet library for
+   inter-locale communication.  GASNet is a one-sided communication
+   and active message layer being developed by Lawrence Berkeley
+   National Lab and UC Berkeley.  For details, refer to:
+
+   * $CHPL_HOME/third-party/gasnet/README and
+   * $CHPL_HOME/third-party/README
+
+
+#. 
+   .. _set-comm-conduit:
+
+   Advanced users can set ``CHPL_COMM_SUBSTRATE`` to indicate the GASNet
+   conduit that they wish to use.  Novice users can leave this unset
+   and Chapel will make a choice for them.  Current defaults are:
+
+     =====================  =====================
+     When compiling for...  Chapel will choose...
```

Comment:
What do you think of doing `CHPL_TARGET_PLATFORM CHPL_COMM_SUBSTRATE` instead of `When compiling for...  Chapel will choose..`?